recipes/julia/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "julia" %}
+{% set version = "0.5.2" %}
+{% set sha256 = "efa3303d63e39bf3ea06e23ef68807cca0095c7e0507bdf26eae1d5dee7868c7" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/JuliaLang/julia/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [osx or win]
+  number: 0
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - gcc  # [linux]
+    - toolchain
+    - patchelf
+    - cmake
+    - zlib 1.2.8
+    - python 2.7.*
+    - perl 5.20.*
+    - openblas 0.2.19|0.2.19.*
+    - blas 1.1 {{ variant }}
+    - fftw
+    - gmp 6.1.*
+    - mpfr 3.1.*
+    - curl
+    - libssh2 1.7.*
+    - libgit2
+
+  run:
+    - zlib 1.2.8
+    - libgcc  # [linux]
+    - libgit2
+    - fftw
+    - gmp 6.1.*
+    - mpfr 3.1.*
+    - openblas 0.2.19|0.2.19.*
+    - blas 1.1 {{ variant }}
```

Comment:
libcurl is also indirect at runtime (for proxies in libgit2 if it's built with that support), though curl or wget is used for downloads at build and test time